CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 ## Changelog
 
+### v1.4.6 Sep 18, 2024
+
+* Upgrade Reader SDK version support to 1.7.7 for iOS
+
 ### v1.4.5 Apr 25, 2024
 
 * Upgrade Reader SDK version support to 1.7.2 for iOS
 
 ### v1.4.4 Aug 16, 2023
 
-* Upgrade Reader SDK version support to 1.7.5 for Android
-* Support CompileSDK and targetSDK 33 on Android
+* Upgrade Reader SDK version support to 1.7.7 for Android
```

Comment:
I don't think you should be updating the previous changelog. All of the changes should be for the newest entry.